MODULE.bazel

```diff
@@ -91,21 +91,9 @@ dev_pip = use_extension(
     dev_dependency = True,
 )
 dev_pip.parse(
-    experimental_requirement_cycles = {
-        "sphinx": [
-            "sphinx",
-            "sphinxcontrib-serializinghtml",
-            "sphinxcontrib-qthelp",
-            "sphinxcontrib-htmlhelp",
-            "sphinxcontrib-devhelp",
-            "sphinxcontrib-applehelp",
-        ],
-    },
     hub_name = "dev_pip",
     python_version = "3.11",
-    requirements_by_platform = {
-        "//docs/sphinx:requirements.txt": "linux_*,osx_*",
-    },
+    requirements_lock = "//docs/sphinx:requirements.txt",
```

Comment:
Now the `requirements` work for all platforms and we can just have a single file, simplifying the usage. The requirements cycles disappeared when updating to the latest versions of the pkgs.

Reply:
Yay! I think we have some stuff in examples that can be switched over to that? (eventually; doesn't have to be in this PR)